### PR TITLE
Add contributing and code of conduct - relates to #49

### PR DESCRIPTION
**Pull request by Lith**
Signed-off-by: Lith <lith@amykron.com>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+    
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
```
Comment (dgmike) [inline CODE_OF_CONDUCT.md:10]: remove extra comma.

```diff
- nationality, personal appearance, race, religion, or sexual identity and
+ nationality, personal appearance, race, religion or sexual identity and
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
```
Comment (dgmike) [inline CODE_OF_CONDUCT.md:1]: remove extra line

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+---
+
+## Pull Request Process
+
+The team will review your pull request and will either merge it, request changes to it, or close it.
+
+**Before submitting your pull request** make sure the following requirements are fulfilled:
+
+1. Fork the repository and create your branch from `master`.
+    - Run `yarn install` in the repository root.
```
Comment (dgmike) [inline CONTRIBUTING.md:29]: It's possible using `npm` too, but... OK

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    ```
+   Git even has a -s command line option to append this automatically to your commit message:
+   ```
+   $ git commit -s -m 'This is my commit message'
+   ```
+4. A team leader will check, suggest modification, approve, merge, and close your request.
+
+## Contribution Prerequisites
+
+* You have [Node](https://nodejs.org/en/) at v10.x.x only and [Yarn](https://yarnpkg.com/en/) at v1.2.0+.
```
Comment (dgmike) [inline CONTRIBUTING.md:51]: Or... `npm` :rofl:

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+---
+
+## Miscellaneous
+
+### Repository Organization
+
+We chose to to use a monorepo design that exploits [Yarn Workspaces](https://yarnpkg.com/en/docs/workspaces) in the way [React](https://github.com/facebook/react/tree/master/packages) or [Babel](https://github.com/babel/babel/tree/master/packages) does. This allows the community to easily maintain the whole ecosystem, keep it up-to-date and consistent.
+
+We do our best to keep the master branch as clean as possible, with tests passing at all times. However, it may happen that the master branch moves faster than the release cycle. Therefore check the [releases on npm](https://www.npmjs.com/package/strapi) so that you're always up-to-date with the latest stable version.
```
Comment (dgmike) [inline CONTRIBUTING.md:62]: `npm` is linking to strapi, is it right?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+**Before submitting your pull request** make sure the following requirements are fulfilled:
+
+1. Fork the repository and create your branch from `master`.
+    - Run `yarn install` in the repository root.
+    - If you’ve fixed a bug or added code that should be tested, add the tests and then link the corresponding issue in either your commit or your PR!
+    - Ensure the test suites are passing:
+      - `yarn test`
+    - Make sure your code lints 
+      - `yarn lint`
```
Comment (dgmike) [inline CONTRIBUTING.md:34]: Is needle to talk about `npm run coverage` or `yarn coverage` in this steps?

File: CONTRIBUTING.md (hunk 3 above)
Comment (Lith) [reply CONTRIBUTING.md:29]: I've reproduce Strapi recommendation and we use `yarn.lock` files for this project.
Using `npm` could generate warning.

File: CONTRIBUTING.md (hunk 5 above)
Comment (Lith) [reply CONTRIBUTING.md:62]: 🤔  I think I need to change this part, I've get this from Strapi contributing guide, I will remove this part.

**Pull request merged**